src/dotnet-scaffolding/dotnet-scaffold-aspnet/Program.cs

```diff
@@ -358,12 +379,21 @@ static void CreateOptions(
         pageTypeOption = new ScaffolderOption<string>
         {
             DisplayName = "Page Type",
-            CliOption = "--page",
+            CliOption = Constants.CliOptions.PageTypeOption,
             Description = "The CRUD page(s) to scaffold",
             Required = true,
             PickerType = InteractivePickerType.CustomPicker,
             CustomPickerValues = BlazorCrudHelper.CRUDPages
         };
+
+        viewsOption = new ScaffolderOption<bool>
+        {
+            DisplayName = "With Views?",
```

Comment:
Just curious: none of the others have the `DisplayName` as a question, but this. Is that correct?

Reply:
`--prerelease` option has it, it looks good in the scaffolding flow, as it is a yes/no question.